SEED-platform/seed
# Performance

#### What's this PR do?
- Updates linux documentation
- Updates js dependencies
- Updates and simplifies ui-grid decorators
- Added ColumnListSettingFilter
- Fixed landing background for ultrawide monitors
- Removed deprecated saving indicator from header
- Consistent trademark usage
- Consistent landing page content
- Fixed long-standing bug where mapping suggestion typeahead would match too many columns.  For instance typing "state" would match every column.
- Prevented autocomplete from overriding typeahead
- Improved UI handling of new org creation
- Replaced custom NoPagination class with better alternative
- Remove notes pagination and fixed notes horizontal scrolling
- Fixed `create_default_user` bug where capital letters in email would prevent login
- Lint fixes
- Added chunked inventory loading
#### How should this be manually tested?
1. `npm install`
2. Monkey test, and check inventory loading for organizations with 5,000+ records
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/411466/68983779-da450a00-07c9-11ea-94e6-7ac039b2fcbf.png)
